fmatch/test_fmatch.py

```diff
@@ -2,35 +2,47 @@
 test_fmatch
 """
 
+from datetime import datetime
 import sys
+import warnings
 # pylint: disable=import-error
 import pandas as pd
 
 # pylint: disable=import-error
 from matcher import Matcher
 
-match = Matcher(index="perf_scale_ci")
-res=match.get_metadata_by_uuid("b4afc724-f175-44d1-81ff-a8255fea034f",'perf_scale_ci')
+warnings.filterwarnings("ignore", message="Unverified HTTPS request.*")
+warnings.filterwarnings(
+    "ignore", category=UserWarning, message=".*Connecting to.*verify_certs=False.*"
+)
+
+match = Matcher(index="perf_scale_ci*", verify_certs=False)
```

Comment:
Do we want to change it to `ospst-*` as well?

Reply:
Currently this utility works on the qe instance...

Reply:
Nope - as that would require having access to internal resources.

Reply:
okay. Understood.